socorro/schemas/processed_crash.schema.yaml

```diff
@@ -505,6 +505,19 @@ definitions:
         description: Process ID of the crashing process.
         type: ["integer", "null"]
         permissions: ["public"]
+      soft_errors:
+        description: >
+          Non-fatal errors encountered by minidump-writer during
+          generation. Extracted from minidump. Useful for debugging Crash
+          Reporters.
+        type: ["array", "null"]
+        items:
+          description: >
+            A serialized Rust structure or string representing a particular
+            type of non-fatal error.
+          type: ["object", "string", "null"]
```

Comment:
I think we want `["array", "null"]`. I don't really know what effect this field has, though.